docs/source/CHANGELOG.rst

```diff
@@ -9,12 +9,16 @@ TornadoVM 1.0.2
 ----------------
 29/02/2024
 
+<<<<<<< HEAD
```

Comment:
It looks like an older file conflict got in the way. Remove this line

Reply:
Done